src/cloudwatch/cloudwatch.go

```diff
@@ -60,6 +65,12 @@ func batchLogEntries(cloudwatchLogsData events.CloudwatchLogsData, channel chan
 				Timestamp: strconv.FormatInt(record.Timestamp, 10),
 				Log:       message,
 			}
+			if strings.HasPrefix(cloudwatchLogsData.LogGroup, common.LambdaLogGroup) {
```

Comment:
nit : I don't think this if check needs to be a part of the for loop since every message is going to have the same log group